app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,16 @@
+<resources>
+  <!-- Default screen margins, per the Android Design guidelines. -->
+  <dimen name="activity_horizontal_margin">16dp</dimen>
+  <dimen name="activity_vertical_margin">16dp</dimen>
+  <dimen name="nav_header_vertical_spacing">8dp</dimen>
+  <dimen name="nav_header_height">176dp</dimen>
+  <dimen name="fab_margin">16dp</dimen>
+  <dimen name="svg_dimention">48dp</dimen>
+  <dimen name="margin_30">30dp</dimen>
+  <dimen name="image_dim_24">24dp</dimen>
+  <dimen name="margin_10">10dp</dimen>
+  <dimen name="margin_8">8dp</dimen>
+  <dimen name="textsize_20">20sp</dimen>
+  <dimen name="imageview_dim_64">64dp</dimen>
+
```

Comment:
Remove this blank line

Reply:
please refer doc for xml formatting guidelines 
https://docs.google.com/document/d/1AHIJagMfATgpbNCFmYoYaSqzsKp8JjTgPLQFXEFJdUA/edit?usp=sharing